docs/roles/integrator/exchange-integration.md

```diff
@@ -26,6 +26,13 @@ sidebar_label: Exchange Integration
   
   const transaction = nearAPI.transactions.createTransaction(signerId, signerPublicKey, receiverId, nonceForPublicKey, actions, blockHash);
   ```
+  - Once your transaction is constructed, you will then need to sign it by calling the `signTransactionObject` method and pass `transaction`, `signerId`, and a `networkId` (i.e. `testnet`, `betanet`, or `mainnet`)
```

Comment:
the network id is purely internal to near-api-js and is used to select the signer. We should not talk about it as if it is some general pattern

Reply:
would just removing the (i.e. `testnet`, `betanet`...) suffice, or would you rather no mention of arguments passed?

Reply:
I think we should just remove the mention of `networkId` or mention that it is specific to near-api-js

Reply:
updated @bowenwang1996 :)